src/js/components/PageHeader.js

```diff
@@ -11,7 +11,7 @@ class PageHeader extends React.Component {
     }
 
     let iconClasses = classNames(
-      'icon icon-large icon-image-container',
+      'icon icon-large icon-app-container icon-image-container',
```

Comment:
I think `icon-app-container` wasn't in there before so this component could be more generic, letting the pages using the pageheaders chose to add either `icon-app-container` or `icon-user-container` etc depending on what kind of icon it needs. Not sure if that makes more sense of if non-`icon-app-container` pages should use `classnames.dedupe` to remove `icon-app-container` and add `icon-{thing}-container`?